etc/config/c++.amazon.properties

```diff
@@ -213,14 +213,26 @@ group.cross.groupName=Cross GCC
 ###############################
 # GCC for PPC
 group.ppc.compilers=ppcg48:ppc64leg630
-group.ppc.groupName=PowerPC GCC
+group.ppc.groupName=POWER Compilers
 group.ppc.isSemVer=true
 compiler.ppcg48.exe=/opt/compiler-explorer/powerpc/gcc-4.8.5/bin/powerpc-unknown-linux-gnu-g++
 compiler.ppcg48.name=PowerPC gcc 4.8.5
 compiler.ppcg48.semver=4.8.5
 compiler.ppc64leg630.exe=/opt/compiler-explorer/powerpc64le/gcc-6.3.0/powerpc64le-unknown-linux-gnu/bin/powerpc64le-unknown-linux-gnu-g++
-compiler.ppc64leg630.name=PowerPC64 gcc 6.3.0
+compiler.ppc64leg630.name=power64le gcc 6.3.0
 compiler.ppc64leg630.semver=6.3.0
+compiler.ppc64leg8.exe=/opt/compiler-explorer/powerpc64le/gcc-at12/powerpc64le-unknown-linux-gnu/bin/powerpc64le-unknown-linux-gnu-g++
+compiler.ppc64leg8.name=power64le at gcc 8 (trunk)
+compiler.ppc64leg8.semver=8
+compiler.ppc64g8.exe=/opt/compiler-explorer/powerpc64/gcc-at12/powerpc64-unknown-linux-gnu/bin/powerpc64-unknown-linux-gnu-g++
+compiler.ppc64g8.name=power64 at gcc 8 (trunk)
+compiler.ppc64g8.semver=8
```

Comment:
This semver should be of the form `8.X.Y`

Reply:
@grooverdan I think this request is the last change needed to get things working?

Reply:
Looked a bit closer, seems I've depended on a newer version of ct-ng than the 0.23.0 in the gcc-cross Docker image. Will revisit and try to get under ct-ng-0.23.0 or install a newer git based version. Building the ct-ng-0.22.0 (old) in the Docker image compile failed in Ubuntu-18.04. Will try to fix that too.

Reply:
I'm just playing with the same thing. I've set up a new Dockerfile pulling the latest git version of ct-ng and build it successfully. With that it seems I can build a number of ARM cross-compilers, including newer ones up to 8.2.

@grooverdan I'm just in the process of forking the repo and testing whether the compiler actually all build (so far only tested 2 Versions). Perhaps it makes sense to combine my attempts with ARM and yours with Power?